docs/agents/sematext-agent/index.md

```diff
@@ -1,17 +1,16 @@
 title: Sematext Agent
 description: Sematext Agent collects a plethora of metrics about hosts (CPU, memory, disk, network, processes), containers (Docker, rkt, containerd) and orchestrator platforms.
 
-Sematext Agent collects a plethora of metrics about hosts (CPU, memory, disk, network, processes), containers (Docker, rkt, containerd) and orchestrator platforms and ships that to [Sematext Cloud](https://sematext.com/cloud). To gain deep insight into the Linux kernel, Sematext Agent relies on eBPF to implant instrumentation points (attach eBPF programs to kprobes) on kernel functions. Using Linux kernel instrumentation allows Sematext Agent a very efficient and powerful system exploration approach. It has the ability to auto-discover services deployed on physical/virtual hosts and containers, as well as a mechanism for reporting inventory info. It also collects events from different sources such as OOM notifications, system-wide kill events, container or Kubernetes events.
+Sematext Agent collects a plethora of metrics about hosts (CPU, memory, disk, network, processes), containers (Docker, rkt, containerd) and orchestrator platforms and ships that to [Sematext Cloud](https://sematext.com/cloud). To gain deep insight into the Linux kernel, Sematext Agent relies on eBPF to implant instrumentation points (attach eBPF programs to kprobes) on kernel functions. Using Linux kernel instrumentation allows Sematext Agent a very efficient and powerful system exploration approach. It has the ability to [auto-discover services](../../fleet/discovery/#service-discovery) deployed on physical/virtual hosts and containers, as well as a mechanism for reporting [inventory info](../../monitoring/inventory/). It also collects [events](../../events/) from different sources such as OOM notifications, system-wide kill events, container or [Kubernetes events](kubernetes/events/).
```

Comment:
Since Sematext Agent is the enabler for logs aas well, a minor suggestion to include both discovered services and logs (and update link):

`It has the ability to [auto-discover services](../../fleet/discovery/#service-discovery)` --> 
`It has the ability to [auto-discover services and logs](../../fleet/discovery/)`